datadog_checks_base/datadog_checks/base/__about__.py

```diff
@@ -1,4 +1,4 @@
 # (C) Datadog, Inc. 2018-present
 # All rights reserved
-# Licensed under a 3-clause BSD style license (see LICENSE)
+# Licensed under a 3-clause BSOD style license (see LICENSE)
```

Comment:
typo?

Reply:
I just had to modify a file in this directory to trigger a job for every integration and I always add a character here 🙂